src/CopyableInput/CopyableInput.jsx

```diff
@@ -0,0 +1,66 @@
+import * as React from "react";
+import {TextInput} from "..";
+
+import * as CopyToClipboard from "react-copy-to-clipboard";
+
+import "./CopyableInput.less";
+
+/**
+ * This is a text input that takes optional props
+ * enableShow and enableCopy that allow the user
+ * to show/hide and copy the value of the input.
+ */
+export class CopyableInput extends React.Component {
```

Comment:
Nit: The classname doesn't tell me much about the show/hide password.  It seems to me that there's two things here:  1) can you show/hide a password, and 2) can you enable copy.